projects/igniteui-angular/src/lib/date-picker/date-picker.component.ts

```diff
@@ -161,7 +163,15 @@ export class IgxDatePickerComponent implements IDatePicker, ControlValueAccessor
      * ```html
      * <igx-date-picker [label]="Calendar"></igx-date-picker>
      * ```
+     * @deprecated By default the label is set to 'Date'. Use
```

Comment:
By default the label is set to 'Date' is not necessary information for this property (the default value is visible). Put something like `Use igxLabel inside the date picker to change the label: <code>`